### PR TITLE
Updated entries

### PR DESCRIPTION
**Pull request by juleoc02**
Updated incomplete entries, moved entries to correct year

### CODE REVIEW

File: publications-2018.bib [hunk 1]
```diff
@@ -915,6 +939,22 @@ @Article{scott18:mta
   doi = {10.1109/TCAD.2018.2789729}
 }
 
+@article{DeVilliers2018,
+author = {de Villiers, A. M. and McBride, A. T. and Reddy, B. D. and Franz, T. and Spottiswoode, B. S.},
+doi = {10.1007/s10237-017-0973-8},
+file = {:home/heister/.local/share/data/Mendeley Ltd./Mendeley Desktop/Downloaded/de Villiers et al. - 2018 - A validated patient-specific FSI model for vascular access in haemodialysis.pdf:pdf},
```
Comment (bangerth) [inline publications-2018.bib:945]: just remove the `file` line

File: publications-2018.bib [hunk 2]
```diff
@@ -1231,17 +1271,6 @@ @Article{lee2018optimal
 publisher={Springer International Publishing}
 }
 
-@Article{lee2018phase,
-title={Phase-field modeling of two phase fluid filled fractures in a poroelastic medium},
```
Comment (bangerth) [inline publications-2018.bib:1235]: Did this entry exist twice, or what was the reason for removing it?

Comment (juleoc02) [reply publications-2018.bib:1235]: Yes. It was listed as both Lee2018 and lee2018phase

**Comment by bangerth** [comment]
You'll probably have to edit this one more time:
```
publications-2018.bib:    author={Uwe Köcher and Markus Bause},
publications-2018.bib:keywords = {Blood flow,Finite elements,Fistula,Fluidstructure interaction,Pre-stress},
ERROR: found non-ASCII characters!
script returned exit code 1
```
In the first line, replace `ö` by `{\"o}`. I suspect that in the second line, the dash is a Unicode dash. Just remove the whole line -- we don't need the keywords.

File: publications-2018.bib [hunk 3]
```diff
@@ -440,16 +472,13 @@ @article{2018_26
...
-   journal = {},
-   note = {Submitted},
-   year    = {2018},
-   volume  = {},
-   pages   = {},
-   url     = {},
-   doi     = {}
+@misc{kcher2018mixed,
+    title={A mixed discontinuous-continuous Galerkin time discretisation for Biot's system},
+    author={Uwe Köcher and Markus Bause},
```
Comment (bangerth) [inline publications-2018.bib:477]: this is the first of these lines

File: publications-2018.bib [hunk 4]
```diff
@@ -915,6 +933,21 @@ @Article{scott18:mta
   doi = {10.1109/TCAD.2018.2789729}
 }
 
+@article{DeVilliers2018,
+author = {de Villiers, A. M. and McBride, A. T. and Reddy, B. D. and Franz, T. and Spottiswoode, B. S.},
+doi = {10.1007/s10237-017-0973-8},
+issn = {16177940},
+journal = {Biomechanics and Modeling in Mechanobiology},
+keywords = {Blood flow,Finite elements,Fistula,Fluidstructure interaction,Pre-stress},
```
Comment (bangerth) [inline publications-2018.bib:941]: that's the second one

File: publications-2018.bib [hunk 5]
```diff
@@ -1258,8 +1280,9 @@ @phdthesis{phdthesis
 	author       = {Filippo Guido Davide Sacco},
 	title        = {A 3D adaptive boundary element method for potential flow with nonlinear Kutta condition},
 	school       = {Politecnico di Milano},
-	year         = {2017},
+	year         = {2018},
 	month        = {4}
+	url = {https://www.politesi.polimi.it/handle/10589/140106},
```
Comment (bangerth) [inline publications-2018.bib:1285]: bibtex is also going to complain about this line: you have a stray comma at the end of the line, and none on the previous one.

**Pull request merged**